server/data/data-hbase/src/main/java/org/apache/james/rrt/hbase/HBaseRecipientRewriteTable.java

```diff
@@ -178,7 +172,7 @@ protected Mappings mapAddress(String user, Domain domain) throws RecipientRewrit
                 }
             }
         }
-        return MappingsImpl.fromRawString(mappings);
+        return MappingsImpl.fromRawString(Optional.ofNullable(mappings).orElse(""));
```

Comment:
empty string is not exactly better.
try `Optional.ofNullable(mappings).map(MappingsImpl::fromRawString).orElse(MappingsImpl.emty())`